Screenbox/Pages/PlayerPage.xaml

```diff
@@ -498,10 +531,8 @@
             Grid.Row="0"
             Grid.RowSpan="3"
             Grid.Column="0"
-            Width="72"
-            Height="72"
-            HorizontalAlignment="Center"
-            VerticalAlignment="Center"
+            Width="96"
+            Height="96"
```

Comment:
The badge looks huge??

Comparing to YouTube player of the same size

![ApplicationFrameHost_2DqcCwOfrJ](https://github.com/user-attachments/assets/a67eb717-82ed-47a7-b1cb-3a8bd213fdc2)
![msedge_GvxOrBRAeh](https://github.com/user-attachments/assets/26155d59-53b6-4e40-b86f-f5dd947d52bf)

Reply:
Increased on the normal state and decreased on the compact overlay state, making for a fairer comparison in my opinion